crates/tests/tests-common/goldenfiles/select_array_composite_field.json

```diff
@@ -12,7 +12,20 @@
         "column": "characters",
         "arguments": {}
       }
-    }
+    },
+    "order_by": {
+      "elements": [
+        {
+          "order_direction": "asc",
+          "target": {
+            "type": "column",
+            "name": "characters",
+            "path": []
+          }
+        }
+      ]
+    },
+    "limit": 1
```

Comment:
We added more data to the table, this was done to keep the previous output for the test.